cloud/endagaweb/templates/dashboard/network_detail/denomination.html

```diff
@@ -0,0 +1,305 @@
+                            </div>
+
+                            <div class="form-group row">
+                                <div class="col-md-12">
+                                    <label for="start_amount" id="ariaLable" class="col-3 col-form-label">
+                                        Validity (days)<span class="aseriskField">*</span>
+                                    </label>
+                                    <div class="input-group col-xs-8">
+                                        <input class="form-control" type="text" id="validity_days" name="validity_days"
+                                               placeholder="Enter validity as number of days" required/>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="form-group row">
+                                <div class="col-md-12">
+                                    <div class="input-group col-xs-8">
+                                        <input type="hidden" id="dnm_id" name="dnm_id" value="0" />
+                                        <input type="submit" id="submit" class="btn btn-primary pull-right" value="Create"/>
+                                        <input type="button" id="cancel" class="btn btn-primary pull-right" value="Cancel" style="display: none;"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% endif %}
+
+    </div>
+
+    <div class='modal fade' id='delete-denom-modal'>
+        <div class='modal-dialog'>
+            <div class='modal-content'>
+                <div class='modal-header'>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                    <h4 class='modal-title'>
+                        Confirmation
+                    </h4>
+                </div>
+                <div class='modal-body'>
+                    <h4>
+                        Are you sure you want to delete "<span id="denom_text"></span>"?
+                    </h4>
+                </div>
+                <div class='modal-footer'>
+                    <button type='button' class='btn btn-default' data-dismiss='modal'>Cancel</button>
+                    <button class='btn btn-primary' type='button' id='delete-confirm'>Confirm</button>
+                    <div id='user-messages-container'></div>
+                </div>
+            </div>
+        </div>
+    </div><!-- /.delete-user-modal -->
+
+{% endblock %}
+{% block js %}
+
+    <script type="text/javascript"
+            src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.11.1/jquery.validate.min.js"></script>
```

Comment:
Fixed in commit # https://github.com/facebookincubator/CommunityCellularManager/pull/39/commits/1367cfdba98194e524b8443953ee30b0c7ab1058
Its part of PR https://github.com/facebookincubator/CommunityCellularManager/pull/39